src/v2i-hub/CDASimAdapter/scripts/send_sim_detected_object_udp.py

```diff
@@ -0,0 +1,89 @@
+
+import socket
+import sys
+import json
+import time
+
+# Script for integration testing CDASimAdapter Time Sync functionality.
+# This python script sends periodic time sync messages to a configurable
+# host and port. To Test the Time Sync functionality of the CDASimAdapter
+# set port to the value of the TIME_SYNC_PORT environment variable.
+#
+# TODO Move this script into a more permanent location
+count_num = 0
+
+def generate_sim_external_object():
+    jsonResult = {
+        "metadata": {
+            "type": "Application",
+            "subtype": "SensorDetectedObject",
+            "timestamp": 123,
+            "isSimulated": True
+        },
+        "payload": {
+            "sensor": {
+            "id": "SomeID",
```

Comment:
also maybe we should rename this field everywhere to sensorId instead of id which may be ambiguous. We can also avoid the nested sensor JSON altogether.

Reply:
```
{ "sensorId": "sensor1",
 "proj_string": "asdlasdkasd",
...
```

Reply:
addressed